docs/releases/5.0.txt

```diff
@@ -430,6 +430,8 @@ Models
 * Django now supports `oracledb`_ version 1.3.2 or higher. Support for
   ``cx_Oracle`` is deprecated as of this release and will be removed in Django
   6.0.
+* The new :class:`~django.db.models.GeneratedField` allows creation of
```

Comment:
In my opinion, GeneratedField is a new feature to be mentioned in the doc section "What's new in Django 5.0"

Reply:
Updated.